<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameObject</class>
 <widget class="QWidget" name="GameObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Overview</string>
    </attribute>
    <widget class="QTextBrowser" name="progressReportText">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>31</y>
       <width>301</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="progressReportLabel">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Progress Report:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="playersList">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>98</y>
       <width>161</width>
       <height>201</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="playerInfoButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>208</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="downButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>158</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="upButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>108</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="turnButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-- Take Turn</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>350</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Status: Okay</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="stageLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>400</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stage:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="stageBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>400</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="hideBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>440</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Hide: </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Notes</string>
    </attribute>
    <widget class="QPlainTextEdit" name="playersText">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>451</width>
       <height>491</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="cardList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="cardInfoList">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>121</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Turns</string>
    </attribute>
    <widget class="QListWidget" name="turnList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>731</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="editTurnButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>490</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
